corehq/apps/cleanup/tasks.py

```diff
@@ -1,10 +1,47 @@
+import os
+import json
+from time import time
 from collections import defaultdict
 
+from celery.schedules import crontab
+from celery.task import periodic_task
+
+from django.core.management import call_command
+
+from corehq.apps.hqwebapp.tasks import mail_admins_async
 from corehq.apps.cleanup.management.commands.fix_xforms_with_undefined_xmlns import \
     parse_log_message, ERROR_SAVING, SET_XMLNS, MULTI_MATCH, \
     CANT_MATCH, FORM_HAS_UNDEFINED_XMLNS
 
 
+UNDEFINED_XMLNS_LOG_DIR = '/tmp/'
+
+
+def json_handler(obj):
```

Comment:
surprisingly our json_handle doesn't handle making sets serializable: https://github.com/dimagi/dimagi-utils/blob/master/dimagi/utils/web.py#L173. just wrote a new one here, though i doubt there would be any bad consequence to adding it in our own handler...